documentation/commands.md

```diff
@@ -136,23 +124,53 @@ and land.
 
 _Syntax -_ `AAL123 i [rwy]`
 
-### Reroute
-_Aliases -_ `reroute`, `rr`
+### STAR
+_Aliases -_ `star`
 
-_Information -_ This command allows you to wipe out the aircraft's current
-route, and assign a new route of your choosing. This is similar to the `route`
-command, but this command will allow you to *change the entire route*, while the
-other is meant for specifying a route to follow to join a later point in the
-aircraft's flight plan. Note that the route uses dot format:
+_Information -_ This command tells the plane to add or change their filed
+Standard Terminal Arrival Route to match the route you specify. This must be
+entered in dotted format, and include the point where the STAR is joined, as
+well as the destination airport, for example: `MLP.GLASR9.KSEA`. See the section
+on rerouting for further detail.
 
->Note: Input data needs to be provided with single dots connecting all procedurally-
-linked points (eg KSFO.OFFSH9.SXC or SGD.V87.MOVER), and all other points that will
-be simply a fix direct to another fix need to be connected with double-dots
-(eg HLI..SQS..BERRA..JAN..KJAN).
+_Syntax -_ `AAL123 star [transition].[STAR name].[airport]`
 
-Full Route Example: `KSEA.HAROB5.ERAVE.Q1.ETCHY..MLBEC.BDEGA2.KSFO`
+## Routing commands
+Instructions on the syntax for editing the aircraft's route are described in the `route` and `reroute` commands.
 
-_Syntax -_ `AAL123 rr [route]`
+### ~~Fix~~
+~~_Aliases -_ `f` / `fix` / `track`~~
+
+~~_Syntax -_ `AAL123 f [fixname]`~~
+*_This command has been deprecated_*
+
+### Hold
+_Aliases -_ `hold`
+
+_Information -_ This command instructs the aircraft to enter a holding
+pattern until further notice. The direction (left/right) may be specified,
+as well as the leg length (in minutes), as well as the fix to hold over.
+But you may also omit those specifications, in which case, the aircraft
+will enter a standard holding pattern over their present position (right
+turns, 1 minute legs). To escape a hold, just set a new heading.
+
+_Parameters -_ Specify the direction of turns during the hold with `right`
+or `left`, the leg length, with `[time]min`, and the fix to hold over
+with simply `[fixname]`. Any combination of these arguments provided in
+any order is acceptable, as long as the command `hold` comes first.
+
+_Syntax -_ `AAL123 hold [fixname] [left|right] [leg_time]min` or `AAL123 hold`
+
+### Proceed Direct
+_Aliases -_ `direct` / `pd` / `dct`
+
+_Information -_ This command instructs the aircraft to go direct to a
+navigational fix, taking a shortcut. For example, if an aircraft is flying
+to fixes [A, B, C], issuing the command "pd B" will cause the aircraft
```

Comment:
Let's extend this to like [A,B,C,D,E,F] or something, and then send them direct D, for example. Because if their route literally is A>B>C, then telling them to go direct to B when they started at A won't change anything.